tmt/tests/test-00-bootc-install.sh

```diff
@@ -36,6 +36,23 @@ if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
     ls -al "$BOOTC_TEMPDIR"
 
     pushd "$TMT_TREE"
+        find /etc/yum.repos.d/*.repo -name tag-repository.repo | xargs -I '{}' cp -v '{}' repos.d/
```

Comment:
Did you mean:

find /etc/yum.repos.d -name tag-repository.repo -exec cp -v '{}' repos.d/ \;

Reply:
I agree that /etc/yum.repos.d/*.repo is overhead, so I have changed it to /etc/yum.repos.d/
The find -exec didn't work for me in Testing Farm in container, so I would rather keep xargs there.

Reply:
I see. You are trying to workaround "set -e". find returns 1 if it cannot find anything.